code/datums/chatmessage.dm

```diff
@@ -459,7 +459,7 @@
 		if(5)
 			return "#[num2hex(c, 2)][num2hex(m, 2)][num2hex(x, 2)]"
 
-/atom/proc/balloon_alert(mob/viewer, text, color = null)
+/atom/proc/balloon_alert(mob/viewer, text, color)
```

Comment:
Do not remove default null value for this parameter

Reply:
aren't arguments null by default anyways

Reply:
Explicit is better than implicit

Reply:
Sure I guess

Reply:
> aren't arguments null by default anyways

Yes, but if a parameter is optional, it should have a default value specified